sources/packages/backend/apps/queue-consumers/src/services/cas-supplier/cas-evaluation-result-processor/cas-evaluation-result-processor.ts

```diff
@@ -25,4 +28,45 @@ export abstract class CASEvaluationResultProcessor {
     evaluationResult: CASEvaluationResult,
     summary: ProcessSummary,
   ): Promise<ProcessorResult>;
+
+  /**
+   * Process the errors that occurred during processing.
+   * @param studentSupplier student supplier information from SIMS.
+   * @param summary current process log.
+   * @param error errors that occurred during processing.
+   * @param partialSupplier partial supplier information.ß
+   * @returns processor result.
+   */
+  async processBadRequestErrors(
+    studentSupplier: StudentSupplierToProcess,
+    summary: ProcessSummary,
+    error: string[],
+    auditUserId: number,
+  ): Promise<ProcessorResult> {
+    summary.warn("A known error occurred during processing.");
+    const now = new Date();
+    const auditUser = { id: auditUserId } as User;
+    try {
+      await this.casSupplierRepo.update(
```

Comment:
Need to adjust the method to optionally receive the supplier-found data if a new site creation fails to comply with the below business AC.
```
Any of these errors should make the Supplier isValid = false and in cases where Supplier found but create Site failed, save any valid Supplier info
```